src/test/java/woowacourse/auth/acceptance/AuthAcceptanceTest.java

```diff
@@ -15,6 +16,7 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.jupiter.api.Assertions.assertAll;
+import static woowacourse.shoppingcart.acceptance.CustomerAcceptanceTest.회원가입;
 
 @DisplayName("인증 관련 기능")
 public class AuthAcceptanceTest extends AcceptanceTest {
```

Comment:
도메인을 제외한 테스트가 아래 문제로 모두 실패하고 있네요. 확인 부탁드립니다.

```
Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Cannot drop "PRODUCT" because "FK_CART_ITEM_TO_PRODUCT" depends on it; SQL statement:
drop table if exists product [90107-200]
```